v-next/core/src/internal/hook-manager.ts

```diff
@@ -9,11 +9,7 @@ import {
 import { HardhatPlugin } from "../types/plugins.js";
 import { LastParameter, Return } from "../types/utils.js";
 
-import { validatePluginNpmDependencies } from "./plugins/plugin-validation.js";
-
 export class HookManagerImplementation implements HookManager {
-  readonly #validatedPlugins = new Set<string>();
```

Comment:
This does not appear to have a usage now that validation happens before the `HookManager` is constructed.